samwise.js

```diff
@@ -1,116 +1,117 @@
 /**
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.